compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/split/constants/JvmArrayTypeConstantsGen.java

```diff
@@ -85,7 +85,10 @@ private void visitArrayTypeInitMethod() {
     }
 
     public String add(BArrayType arrayType) {
-        return arrayTypeVarMap.computeIfAbsent(arrayType, str -> generateBArrayInits(arrayType));
+        if (!arrayTypeVarMap.containsKey(arrayType)) {
+            arrayTypeVarMap.put(arrayType, generateBArrayInits(arrayType));
+        }
+        return arrayTypeVarMap.get(arrayType);
     }
```

Comment:
Since `generateBArrayInits(arrayType)`, `generateTypeRefInits(type)`, `generateBTupleInits(type, symbolTable)`, and `generateBUnionInits(type, symbolTable)` do not return `null`, you can write like this.
```java
public String add(BArrayType arrayType) {
        String arrayTypeVarName  = arrayTypeVarMap.get(arrayType);
        if (arrayTypeVarName == null) {
            arrayTypeVarName = generateBArrayInits(arrayType);
            arrayTypeVarMap.put(arrayType, arrayTypeVarName);
        }
        return arrayTypeVarName;
    }
```
Same for the other changes also.